### ISSUE TITLE
Support for Prezto

### ISSUE THREAD

**Issue by haggen** [wbinglee/zsh-wakatime#12]
Is it possible to use this plugin (maybe with a few changes) with [prezto](https://github.com/sorin-ionescu/prezto)?

**Comment by wbinglee** [comment]
I suppose you can simply use it as a module in prezto,  probably changing a script name by prezto convention.

**Comment by haggen** [comment]
This:

```
precmd_functions+=(send_wakatime_heartbeat)
```

Had to be changed to:

```
add-zsh-hook precmd send_wakatime_heartbeat
```

Also the script has to be named `init.zsh`, and the directory must live in `~/.zprezto/modules/`.

I'm not very proficient with zsh but I manage to get it working after these changes.

**Comment by wbinglee** [comment]
Thanks. It used to `add-zsh-hook precmd send_wakatime_heartbeat` but change to `precmd_functions+=(send_wakatime_heartbeat)` due to `add-zsh-hook` doesn't work in none OS X platform. 

I would create a new repo just fore prezto.

**Comment by haggen** [comment]
I'm actually using OS X El Capitan and it's working with `add-zsh-hook`.

**Comment by wbinglee** [comment]
yes, the `add-zsh-hook` on OS X is ok, but looks not work in other platforms like ubuntu, I haven't investigated to see why. I'm using OS X so little bit lazy to do so.

Actually I don't know why `precmd_functions` doesn't work for prezto, as basically the `add-zsh-hook` is just a nicer method to produce `precmd_functions` and others.
